go-controller/pkg/ovn/secondary_layer3_network_controller.go

```diff
@@ -338,6 +341,11 @@ func (oc *SecondaryLayer3NetworkController) newRetryFramework(
 // Start starts the secondary layer3 controller, handles all events and creates all needed logical entities
 func (oc *SecondaryLayer3NetworkController) Start(ctx context.Context) error {
 	klog.Infof("Start secondary %s network controller of network %s", oc.TopologyType(), oc.GetNetworkName())
+
+	if err := oc.ensureNetworkID(); err != nil {
+		return fmt.Errorf("failed ensuring network id at user defined layer3 network controller for network '%s': %w", oc.GetNetworkName(), err)
+	}
```

Comment:
ditto; this sounds like initializing to me ...

Reply:
ditto

Reply:
see https://github.com/ovn-org/ovn-kubernetes/pull/4529#discussion_r1679415103

Reply:
moving there.